### PR TITLE
Slc status final branch

### PR DESCRIPTION
**Pull request by seedyjahateh**
We added the status of slc's when on view. The status changes dynamically based on the status of the proposal.

Fixing issue #1171.

### CODE REVIEW

**Comment by Ndiayem14** [comment]
We added the status of slc's when on view. The status changes dynamically based on the status of the proposal.

File: app/controllers/serviceLearning/routes.py [hunk 1]
```diff
@@ -54,6 +54,7 @@ def slcEditProposal(courseID):
     if g.current_user.isCeltsAdmin or g.current_user in courseInstructors or isCourseCreator:
         course = Course.get_by_id(courseID)
         courseStatus = CourseStatus.get_by_id(course.status)
+        print("Type of courseStatus:", type(courseStatus))
```
Comment (BrianRamsay) [inline app/controllers/serviceLearning/routes.py]: Don't leave debug output in the final code

File: app/models/course.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 from app.models.courseStatus import CourseStatus
 from app.models.note import Note
 from app.models.user import User
+from peewee import CharField, FloatField, BooleanField, ForeignKeyField, TextField
```
Comment (BrianRamsay) [inline app/models/course.py]: These imports should not be necessary. These symbols are already imported in app.models

File: app/templates/serviceLearning/slcProposal.html [hunk 3]
```diff
@@ -198,3 +205,7 @@ <h2 class="text-center pt-4 pb-2"> New Proposal For Service-Learning Course</h2>
   </div>
   <input value={{course.id}} name="courseID" hidden/>
 </div>
+
+<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
```
Comment (BrianRamsay) [inline app/templates/serviceLearning/slcProposal.html]: You definitely didn't need these just to show the status. Where did they come from?

File: app/templates/serviceLearning/slcProposal.html [hunk 4]
```diff
@@ -1,4 +1,11 @@
-<h2 class="text-center pt-4 pb-2"> New Proposal For Service-Learning Course</h2>
+ <h2 class="text-center pt-4 pb-2">Proposal For Service-Learning Course</h2>
+
+      <h3 class="text-center pb-4">Proposal Status:
+        <span style="color: rgb(12, 8, 245);">
+                {{ courseStatus.status }}
+            </span>
```
Comment (BrianRamsay) [inline app/templates/serviceLearning/slcProposal.html]: We don't need a color unless it is going to change for different statuses

File: app/templates/serviceLearning/slcProposal.html [hunk 5]
```diff
@@ -1,4 +1,11 @@
-<h2 class="text-center pt-4 pb-2"> New Proposal For Service-Learning Course</h2>
+ <h2 class="text-center pt-4 pb-2">Proposal For Service-Learning Course</h2>
+
+      <h3 class="text-center pb-4">Proposal Status:
```
Comment (BrianRamsay) [inline app/templates/serviceLearning/slcProposal.html]: "Proposal Status" is redundant. "Status" should be fine.

File: app/models/courseStatus.py [hunk 6]
```diff
@@ -1,4 +1,7 @@
-from app.models import*
+# from app.models import*
+from app.models import baseModel
+from peewee import CharField
```
Comment (BrianRamsay) [inline app/models/courseStatus.py]: We can leave the previous import statement as is.

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

File: app/controllers/serviceLearning/routes.py (hunk 1 above)
Comment (seedyjahateh) [reply app/controllers/serviceLearning/routes.py]: Removed

File: app/models/course.py (hunk 2 above)
Comment (seedyjahateh) [reply app/models/course.py]: Removed.

File: app/templates/serviceLearning/slcProposal.html (hunk 3 above)
Comment (seedyjahateh) [reply app/templates/serviceLearning/slcProposal.html]: Removed.

File: app/templates/serviceLearning/slcProposal.html (hunk 5 above)
Comment (seedyjahateh) [reply app/templates/serviceLearning/slcProposal.html]: Changed.

File: app/models/courseStatus.py (hunk 6 above)
Comment (seedyjahateh) [reply app/models/courseStatus.py]: Now using 'from app.models import*' only.

File: app/templates/serviceLearning/slcProposal.html (hunk 4 above)
Comment (seedyjahateh) [reply app/templates/serviceLearning/slcProposal.html]: removed

**Review by BrianRamsay — APPROVED** [review]

**Pull request merged**